performance_manager/lib/postgres_utils.py

```diff
@@ -150,7 +154,11 @@ def get_local_engine(
         )
 
         engine = sa.create_engine(
-            database_url, echo=echo, future=True, pool_pre_ping=True
+            database_url,
+            echo=echo,
+            future=True,
+            pool_recycle=300,
+            pool_pre_ping=True,
```

Comment:
added `pool_recycle` which is supposed to recycle the connections after set number of seconds, this should force the engine to update the rds password and avoid using an expired auth token